pkg/cmd/runs.go

```diff
@@ -13,41 +13,47 @@ type RunsCmdValues struct {
 	bootstrap string
 }
 
-func createRunsCmd(factory Factory, parentCmd *cobra.Command, rootCmdValues *RootCmdValues) (*cobra.Command, error) {
-	var err error = nil
+type RunsCommand struct {
+	cobraCommand *cobra.Command
+	values       *RunsCmdValues
+}
+
+func NewRunsCmd(factory Factory, rootCommand GalasaCommand) (GalasaCommand, error) {
+	cmd := new(RunsCommand)
+	err := cmd.init(factory, rootCommand)
+	return cmd, err
+}
+
+func (cmd *RunsCommand) GetName() string {
+	return COMMAND_NAME_RUNS
+}
+
+func (cmd *RunsCommand) GetCobraCommand() *cobra.Command {
+	return cmd.cobraCommand
+}
+
+func (cmd *RunsCommand) GetValues() interface{} {
+	return cmd.values
+}
+
```

Comment:
section/region comments to break it up ?

Reply:
doesn't seem to consistently work accross devices